web/sum.js

```diff
@@ -0,0 +1,6 @@
+// Jest Example File
+
+function sum(a, b) {
```

Comment:
I see it's for test propouse but we can keep all logic related files in the src folder